checker/tests/fenum/UpperBoundsInByteCode.java

```diff
@@ -1,3 +1,5 @@
+// @above-java17-jdk-skip-test TODO: reinstate on JDK 18, false positives may be due to issue #979
```

Comment:
Should I look into these warnings?

Reply:
Yes, please look into them as part of your work on #979, when you return to that task.

I don't have an opinion whether you should look into these problems first, or do other parts of the task that you know are essential before seeing whether these get fixed as a side effect.